apps/meteor/.meteorMocks/index.js

```diff
@@ -0,0 +1,5 @@
+import sinon from 'sinon';
```

Comment:
Why this file is not a typescript? We are not adding JS files anymore 🤔

Reply:
oopss, my mistake! :grimacing: fixed